source/snes9x/memmap.h

```diff
@@ -295,11 +295,11 @@ struct CMemory
 	void	ParseSNESHeader (uint8 *);
 	void	InitROM (void);
 
-	uint32	map_mirror (uint32, uint32);
+	uint32 	map_mirror (uint32 size, uint32 pos);
 	void	map_lorom (uint32, uint32, uint32, uint32, uint32);
 	void	map_hirom (uint32, uint32, uint32, uint32, uint32);
 	void	map_lorom_offset (uint32, uint32, uint32, uint32, uint32, uint32);
-	void	map_hirom_offset (uint32, uint32, uint32, uint32, uint32, uint32);
+	void 	map_hirom_offset (uint32 bank_s, uint32 bank_e, uint32 addr_s, uint32 addr_e, uint32 size, uint32 offset);
```

Comment:
Why change these header definitions? Do these changes somehow fix something?

Reply:
@dborth I backported this from:
https://github.com/bubble2k16/snes9x_3ds/commit/95b1f0c03144b8f9161459b1dd4a52b10a2d27aa

It seems that these header definitions were already used in memmap.cpp butnot defined. 

I wanted to backport the code in memmap.cpp but it was already there but not in memmap.h thats the reason I put it in. 

Didn't test the game without this yet.